### PR TITLE
[zh-tw] translate footer

### PR DESCRIPTION
**Pull request by Dr-XYZ**

### CODE REVIEW

File: files/zh-tw/web/html/element/footer/index.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+---
+title: "<footer>: 頁尾元素"
+slug: Web/HTML/Element/footer
```
Comment (JasonLamv-t) [inline files/zh-tw/web/html/element/footer/index.md:3]:
```suggestion
title: "<footer>: 頁尾元素"
slug: Web/HTML/Element/footer
l10n:
  sourceCommit: f19d6973b0e2f5108c1d53e45ad6f18ac61e0fd2
```

**Review by JasonLamv-t — COMMENTED** [review]
LGTM.

**Comment by yin1999** [comment]
@Dr-XYZ, I will review this PR in a few days, and before that I will do some fixes to the relevant documentation.

**Comment by yin1999** [comment]
Hey @Dr-XYZ. May I ask why you closed this PR?

**Comment by yin1999** [comment]
Hey @Dr-XYZ. I'm not sure is there a "restore branch" link at the bottom of the page. If there is one, please click it, and then "Reopen" this PR. Or you need to create a new branch with a new PR being created.

**Comment by Dr-XYZ** [comment]
I open new PR, sorry about that

**Pull request closed without merging**